### PR TITLE
Update cli-reference.mdx

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description (required)

Update cli-reference.mdx

- Add `create-key` command for cli help info
- Delete `--open` flag from cli help info. It's not in astro cli output help info.
- Move `--open` flag to Common flags from Global flags
- Add a section for how to get a list of flags for any command

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Review by ematipico — APPROVED** [review]

File: src/content/docs/en/reference/cli-reference.mdx [hunk 1]
```diff
@@ -80,10 +81,47 @@ Global Flags
...
+The following message will display in your terminal:
+
+```bash
+astro dev [...flags]
+
+Flags
+                 --port  Specify which port to run on. Defaults to 4321.
+                 --host  Listen on all addresses, including LAN and public addresses.
+--host <custom-address>  Expose on a network IP address at <custom-address>
+                 --open  Automatically open the app in the browser on server start
```
Comment (sarah11918) [inline src/content/docs/en/reference/cli-reference.mdx:120]:
```suggestion
                 --host  Listen on all addresses, including LAN and public addresses.
                 --host <custom-address>  Expose on a network IP address at <custom-address>
                 --open  Automatically open the app in the browser on server start
```
Just aligning!

File: src/content/docs/en/reference/cli-reference.mdx [hunk 2]
```diff
@@ -80,10 +81,47 @@ Global Flags
...
+                --force  Clear the content layer cache, forcing a full rebuild.
+            --help (-h)  See all available flags.
```
Comment (sarah11918) [inline src/content/docs/en/reference/cli-reference.mdx:122]:
```suggestion
                --force  Clear the content layer cache, forcing a full rebuild.
                --help (-h)  See all available flags.
```
Same here!

**Review by sarah11918 — APPROVED** [review]
Thank you for your patience @liruifengv !!

Just fixed some spacing below, but otherwise, this PR can be merged whenever you're happy with it!

**Review by casungo — APPROVED** [review]
LGTM!

**Comment by liruifengv** [comment]
!coauthor

**Pull request merged**